### ISSUE TITLE
You can pour reagents directly into the chem master buffer, which causes issues.

### ISSUE THREAD

**Issue by MistakeNot4892** [NebulaSS13/Nebula#2349]
#### Description of issue
Copying from downstream: https://github.com/ScavStation/ScavStation/issues/336
TLDR: You can pour chems into the buffer directly due to the chemmaster being an open container, which breaks.

#### Difference between expected and actual behavior
You should not be able to pour chems in without using a beaker that you insert.

#### Steps to reproduce
- Pick up bucket or bottle.
- Click on chemmaster.

#### Length of time in which bug has been known to occur
This was reported a very long time ago and I don't think anyone has looked at it.

#### Issue bingo
- [X] Issue could be reproduced at least once
- [X] Issue could be reproduced by different players
- [X] Issue could be reproduced in multiple rounds
- [ ] Issue happened in a recent (less than 7 days ago) round
- [X] [Couldn't find an existing issue about this](https://github.com/NebulaSS13/Nebula/issues)

**Comment by Gaxeer** [comment]
I could not reproduce it

**Comment by MistakeNot4892** [comment]
Reproduced it using a booze bottle, I think the beakers etc. just trigger the attackby guards:
![image](https://user-images.githubusercontent.com/2468979/170620633-db808586-953b-4eef-9417-9f23c9dcf576.png)
